### ISSUE TITLE
Name property is not provided for "Clear All" button present besides to 'Tasks' heading: A11y_.NET 9 template_Home_Screen reader

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#27424]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG4.1.2; #A11ySev2; #iOS; #SH_.NET9template_iOS_Jan2025; #Voiceover; #.NET Core; #Device; #screen reader; #NameRoleValue; #AINotInscope; #iphone;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
iOS Version: 18.2
Screen reader: Voiceover

### Repro Steps:

1. Turn on Voiceover and Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Swipe till "Clear all" button present besides to 'Tasks' heading.
4. Observe that Name property is not provided for "Clear All" button present besides to 'Tasks' heading.

### Actual Result:
Name property is not provided for "Clear All" button present besides to 'Tasks' heading

### Expected Result:
Appropriate name property should be provided for "Clear All" button present besides to 'Tasks' heading

### User Impact:
Users who rely on screen reader won't be able to understand the purpose of the button if name property is not provided for it.

### Attachment:

https://github.com/user-attachments/assets/752f8fe7-0605-42b9-880f-766247882ab5

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by PoojaNamde** [comment]
@PureWeen Could you please confirm if the "Clear all button" in the task section has been removed? We need to verify this so that we can close the bug, as we are unable to see that option on the screen.

**Comment by tj-devel709** [comment]
Hi @PoojaNamde, the "Clean Tasks" button is still there and the screen reader is reading it out.

**Comment by merriemcgaw** [comment]
@tj-devel709 / @PureWeen - does this just need to be retested by @PoojaNamde with the lastest?

**Comment by tj-devel709** [comment]
@merriemcgaw I was just in communication with Pooja regarding this one. It's not appearing fixed for her but is for me so I asked if she is testing the latest build in the test flight. I have been transferred to another team so Shane would be the best contact for now going forward!

**Comment by merriemcgaw** [comment]
@tj-devel709  - understood. Thanks for all the updates and best of luck on the new team.
@PureWeen  if there's anything I can do to help here, just say the word!

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:07-28-25; #A11yFixed;

Verified the issue in the latest env issue is fixed hence closing the bug.

![Image](https://github.com/user-attachments/assets/7752cf48-d913-45b8-95cd-ba1c7f00cab0)
